lib/hardware/modules/iob_clock/iob_clock.py

```diff
@@ -0,0 +1,38 @@
+def setup(py_params_dict):
+    attributes_dict = {
+        "original_name": "iob_clock",
+        "name": "iob_clock",
+        "version": "0.1",
+        "confs": [
+            {
+                "name": "CLK_PERIOD",
+                "type": "P",
+                "val": "10",
+                "min": "",
+                "max": "",
+                "descr": "Clock period",
+            },
+        ],
+        "ports": [
+            {
+                "name": "clk",
+                "descr": "Output clock",
+                "signals": [
+                    {"name": "clk", "width": "1", "direction": "output"},
+                ],
+            },
+        ],
+        "snippets": [
+            {
+                "outputs": ["clk_o"],
```

Comment:
what is this for?

Reply:
I needed to declare `clk` as a reg. It seems that's not possible to do on the Python scripts after asking Artur. So instead I declared it on code and assigned the output to it.

Reply:
The snippets 'outputs' list will be used to generate 'global signals', as described here: https://github.com/IObundle/py2hwsw/blob/main/py2hwsw/scripts/iob_core.py#L172

For now, the 'global signals' are not implemented, so that list is not yet used.